.storybook/components/CodeViewer/CodeViewer.helpers.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import reactElementToJSXString from "react-element-to-jsx-string";
+
+// modified getReactElementDisplayName from react-element-to-jsx-string
+// https://github.com/algolia/react-element-to-jsx-string/blob/master/src/parser/parseReactElement.js#L15-L20
```

Comment:
Suggest using a permalink (to get one, open this current link, press `y` and the address bar will have the permalink URL)